CastleBoy/map.cpp

```diff
@@ -115,13 +115,12 @@ void Map::init(const uint8_t* source)
   }
 }
 
-// FIXME can we use uint8_t instead of int16_t ?
```

Comment:
Oh sorry for this bad unclear FIXME.. This was about uint8_t for the local variables tx1,tx2,ty1,ty2,ix and iy. As the CPU of the Arduboy is mostly 8 bit, using unsigned byte is very fast and helps reduce flash size. This is kind of an over-optimization. But I left the FIXME in case one day we run out of flash (or CPU..).

For the signature of the method we better keep it like before (int16_t for x, int8_t for y) as those are the standard types for coordinates in CastleBoy. Unless you had a reason I didn't spotted for that change, I think we better revert it in order to keep consistency accross the CastleBoy source.